js/main.js

```diff
@@ -1,7 +1,46 @@
-
 // so that we can use anywhere, will be used for pretty much everything
 let storage = window.localStorage;
 
+document.addEventListener("DOMContentLoaded", function() {
+  loadTodos();
+
+  if (isUsernameSet()) {
+    loadUsername()
+  } else {
+    let usernameElement = document.querySelector(".username h1 span")
+    usernameElement.textContent = "User"
```

Comment:
When I loaded the page the first time it just said "Hello User", there was no form visable on firs time setup. So I found it a bit unintuitive, I guessed that doubleclick might do something. IMO when loading without having set your first username the form should be visable. 

And I like that you can doubleclick it and edit your name!

Reply:
hmm I forgot that. Will fix